api/types.go

```diff
@@ -727,3 +727,9 @@ func parseAssociationTargets(rawTgts []*ssm.Target) (tgts []AssociationTarget) {
 type Ec2InstanceStateChangeRequest struct {
 	State string
 }
+
+type SSMSendCommand struct {
```

Comment:
Let's call this `SsmCommandRequest`
We've been trying to use Request for all of the API request inputs.

Reply:
Done.